docs/en/ingest-management/release-notes/release-notes-7.14.asciidoc

```diff
@@ -86,6 +86,38 @@ malicious users.
 
 ====
 
+[[known-issue-938]]
+.Failure in default policy setup may result in policy with no input 
+[%collapsible]
+====
+
+*Details*
+
+During the setup of default policies, an error may occur, such as a network
+error or connectivity issue, that results in a default policy with no input.
+
+*Impact* +
+
+To resolve this problem, you must:
+
+. Delete the default {fleet-server} policy saved object.
+
+. Go to the {fleet} page in {kib} to force reloading of the object.
+
+//QUESTION: Was there a typo in issue 938, or is it really the default
```

Comment:
Yes I mean the default agent policy saved object used by Fleet server, when we setup Fleet we create two default agent policy one with the system integration one with the fleet server one.